uint16 voltage
uint8  control_mode
uint8  status
uint8  drive_motor_error
uint8  encode_error
